apps/fabric-website-resources/package.json

```
@@ -22,6 +22,7 @@
     "code-style": "node ../../scripts/just.js code-style",
     "start": "node ../../scripts/just.js dev",
     "start-test": "node ../../scripts/just.js jest-watch",
+    "jest-dom": "node ../../scripts/just.js jestDom",
```

Based on the task definition later I think this should be:
```suggestion
    "jest-dom": "node ../../scripts/just.js jest-dom",
```

---

👍